doc/lib.rst

```diff
@@ -88,10 +91,10 @@ Algorithms
 ----------
 
 * `algorithm <algorithm.html>`_
-  This module implements some common generic algorithms like sort or binary search.
+  This module implements some common generic algorithms on `openArray`s.
 
 * `enumutils <enumutils.html>`_
-  This module adds functionality for the built-in `enum` type.
+  This module adds functionality for `enum`s.
```

Comment:
The old desc had a little bit more info and was better, IMHO.

Reply:
`enum` itself isn't a type though, I don't see how "the builtin `enum` type" makes sense.

Reply:
Good point, oh well.